src/dvc_objects/fs/__init__.py

```diff
@@ -96,9 +92,7 @@ def __init__(self, reg):
         self._registry = reg
 
     def __getitem__(self, key):
-        entry = self._registry.get(key)
-        if not entry:
-            return LocalFileSystem
+        entry = self._registry.get(key) or self._registry[Schemes.LOCAL]
```

Comment:
I changed this so that `LocalFileSystem` get imported in the same way as other filesystems do.